tests/kafkatest/services/kafka/kafka.py

```diff
@@ -435,8 +435,8 @@ def create_topic(self, topic_cfg, node=None):
         kafka_topic_script = self.path.script("kafka-topics.sh", node)
 
         cmd = kafka_topic_script + " "
-        cmd += "--zookeeper %(zk_connect)s --create --topic %(topic)s " % {
-                'zk_connect': self.zk_connect_setting(),
+        cmd += "--bootstrap-server %(bootstrap_server)s --create --topic %(topic)s " % {
```

Comment:
Currently we are not passing required security configs (using --command-config) to the tool.  This change may not work for with secure broker listeners.  seems like we are not using these methods in any security enabled tests.

Reply:
Good catch!  I fixed it by having each Kafka broker create a (potentially empty, for `PLAINTEXT`) `command_config.properties` file whenever it starts; every command that uses `--bootstrap-server` also now sets `--command-config` and exports any necessary SASL-related system properties.  I successfully ran all three tests `zookeeper_tls_encrypt_only_test`, `zookeeper_tls_test`, and `zookeeper_security_upgrade_test`.  I think the full system test suite should pass with these changes.